### PR TITLE
publiccloud: Add AZURE_LOGIN_WAIT_SECONDS to handle hiccups happening…

### PR DESCRIPTION
**Pull request by asmorodskyi**
Add AZURE_LOGIN_WAIT_SECONDS to handle hiccups happening with very new password

### CODE REVIEW

File: lib/publiccloud/azure.pm [hunk 1]
```diff
@@ -57,6 +57,12 @@ sub az_login {
     my $login_cmd = sprintf(q(while ! az login --service-principal -u '%s' -p '%s' -t '%s'; do sleep 10; done),
         $self->key_id, $self->key_secret, $self->tenantid);
     assert_script_run($login_cmd, timeout => 5 * 60);
+    #Azure infra need some time to propagate given by Vault credentials
+    # Running some verification command does not prove anything because
+    # at the beginning failures can happening sporadically
+    if (get_var('AZURE_LOGIN_WAIT_SECONDS')) {
+        sleep(get_var('AZURE_LOGIN_WAIT_SECONDS'));
+    }
```
Comment (cfconrad) [inline lib/publiccloud/azure.pm:65]:
```suggestion
    sleep(get_var('AZURE_LOGIN_WAIT_SECONDS',0));
```

**Comment by asmorodskyi** [comment]
VR: http://autobot/tests/634

File: lib/publiccloud/azure.pm (hunk 1 above)
Comment (asmorodskyi) [reply lib/publiccloud/azure.pm]: done

**Comment by asmorodskyi** [comment]
> VR: http://autobot/tests/634

run failed due to current issues with github , please not merge before I will provide new VR

**Comment by asmorodskyi** [comment]
VR: http://autobot/tests/635

**Pull request merged**